src/ngSanitize/sanitize.js

```diff
@@ -200,6 +200,44 @@ function $SanitizeProvider() {
     }
   };
 
+
+  /**
+   * @ngdoc method
+   * @name $sanitizeProvider#addValidElements
+   * @kind function
+   *
+   * @param {Array|Object=} elements List of valid elements.
+   */
+  this.addValidElements = function(elements) {
+    var _validElements = elements;
+
+    if (angular.isObject(elements)) {
+      _validElements = [];
+      angular.forEach(['voidElements', 'inlineElements', 'blockElements', 'svgElements'], function(elementsType) {
+        if (elements[elementsType]) {
+          if (elementsType === 'voidElements') {
+            angular.extend(voidElements, toMap(elements.voidElements.join(',')));
+          }
+          _validElements = _validElements.concat(elements[elementsType]);
```

Comment:
Since we treat most categories (except for `voidElements` and `svgElements`) the same, I am wondering if it would be more striaght forward to ue a simpler public API, e.g.:

```
{
  htmlElements: [...],
  htmlVoidElements: [...],
  svgElements: [...],
}
```

Reply:
Looks good to me